Tools/autotest/sim_vehicle.py

```diff
@@ -1055,8 +1055,27 @@ def generate_frame_help():
                  help="disable mavproxy rcin")
 parser.add_option_group(group)
 
+group_completion = optparse.OptionGroup(parser, "Completion helpers")
+group_completion.add_option("", "--list-vehicle",
+                            action='store_true',
+                            help="List the vehicles")
+group_completion.add_option("", "--list-frame",
+                            type='string',
+                            default=None,
+                            help="List the vehicle frames")
+parser.add_option_group(group_completion)
+
 cmd_opts, cmd_args = parser.parse_args()
 
+if cmd_opts.list_vehicle:
+    print(' '.join(vinfo.options.keys()))
+    sys.exit(1)
+if cmd_opts.list_frame:
```

Comment:
Won't this mean we'll complete frames for incorrect vehicles?

I wonder if this should be `--list-all-frames` - so maybe later we can `--list-frames` and base the results off whether a vehicle has been specified...

Reply:
Normally, this expect a vehicle type to output only the vehcile type frame
I can add a check if the vehicle type is unknown, but it should be already covered